mteb/tasks/Classification/eng/LegalBenchClassification.py

```diff
@@ -2916,3 +2916,528 @@ def dataset_transform(self):
+
+        self.dataset = self.dataset.map(
+            lambda example: {
+                "text": example["text"]
+                + "\n\n"
+                + "Parties are diverse: "
+                + example["parties_are_diverse"]
+                + "\n\n"
+                + "Amount-in-controversy is met: "
+                + example["aic_is_met"]
+            }
+        )
+        self.dataset = self.dataset.remove_columns(
+            ["parties_are_diverse", "aic_is_met"]
+        )
+
+
+class FunctionOfDecisionSectionLegalBenchClassification(AbsTaskClassification):
+    metadata = TaskMetadata(
+        name="FunctionOfDecisionSectionLegalBenchClassification",
+        description="Classify the function of different sections of legal written opinions.",
+        reference="https://huggingface.co/datasets/nguha/legalbench",
+        dataset={
+            "path": "nguha/legalbench",
+            "name": "function_of_decision_section",
+            "revision": "12ca3b695563788fead87a982ad1a068284413f4",
+        },
+        type="Classification",
+        category="s2s",
+        eval_splits=["test"],
+        eval_langs=["eng-Latn"],
+        main_score="accuracy",
+        date=("2000-01-01", "2023-08-23"),  # best guess
+        form=["written"],
+        domains=["Legal"],
+        task_subtypes=[],
+        license="cc-by-4.0",
+        socioeconomic_status="high",
+        annotations_creators="expert-annotated",
+        dialect=[],
+        text_creation="found",
+        bibtex_citation="""
+        @misc{guha2023legalbench,
+            title={LegalBench: A Collaboratively Built Benchmark for Measuring Legal Reasoning in Large Language Models}, 
+            author={Neel Guha and Julian Nyarko and Daniel E. Ho and Christopher Ré and Adam Chilton and Aditya Narayana and Alex Chohlas-Wood and Austin Peters and Brandon Waldon and Daniel N. Rockmore and Diego Zambrano and Dmitry Talisman and Enam Hoque and Faiz Surani and Frank Fagan and Galit Sarfaty and Gregory M. Dickinson and Haggai Porat and Jason Hegland and Jessica Wu and Joe Nudell and Joel Niklaus and John Nay and Jonathan H. Choi and Kevin Tobia and Margaret Hagan and Megan Ma and Michael Livermore and Nikon Rasumov-Rahe and Nils Holzenberger and Noam Kolt and Peter Henderson and Sean Rehaag and Sharad Goel and Shang Gao and Spencer Williams and Sunny Gandhi and Tom Zur and Varun Iyer and Zehua Li},
+            year={2023},
+            eprint={2308.11462},
+            archivePrefix={arXiv},
+            primaryClass={cs.CL}
+        },
+        """,
+        n_samples={"test": 367},
+        avg_character_length={"test": 551.07},
+    )
+
+    def dataset_transform(self):
+        self.dataset = self.dataset.rename_column("answer", "label")
+
+        self.dataset = self.dataset.map(
+            lambda example: {
+                "text": example["Paragraph"]
+                + "\n\n"
+                + "Citation: "
+                + example["Citation"]
```

Comment:
Not entirely sure what is being classified here so unsure if this format makes sense

Reply:
I think this format is fine here, since the goal is to classify the paragraph and the citation is just an added reference. 

An example from this dataset after formatting would look like this:
```
We need go no further. For aught that appears, the appellant was fairly tried before an able and thoughtful judge and an impartial jury, justly convicted, and lawfully sentenced. For the reasons elucidated above, the judgment of the district court is Affirmed.

Citation: F.4th 100
```

Reply:
Perfect I think that it totally fine given the task